f5/bigip/resource.py

```diff
@@ -575,7 +576,11 @@ def _create(self, **kwargs):
         # Post-process the response
         self._local_update(response.json())
 
-        if self.kind != self._meta_data['required_json_kind']:
+        kinds = []
+        kinds.append(self._meta_data['required_json_kind'])
+        kinds.append("tm:transaction:commandsstate")
```

Comment:
no
f5.bigip.resource.KindTypeMismatch: For instances of type ''Pool'' the corresponding kind must be ''tm:ltm:pool:poolstate'' but creation returned JSON with kind: u'tm:transaction:commandsstate'